ci-operator/config/openshift/installer/openshift-installer-main.yaml

```diff
@@ -1296,6 +1296,19 @@ tests:
         AGENT_MINIMAL_ISO=true
     workflow: agent-e2e-generic
   timeout: 6h0m0s
+- as: e2e-agent-iso-no-registry-compact-ipv4
```

Comment:
It could be useful also in 4.20 / 4.21 (the last just to avoid losing it at the next branch cut).
Given also that this kind of job is becoming pretty relevant, I think it may be nice to have a workflow for it, to avoid config duplication across different repos